nd4j/nd4j-parameter-server-parent/nd4j-parameter-server-node/src/main/java/org/nd4j/parameterserver/distributed/v2/enums/TransmissionStatus.java

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2018 Skymind, Inc.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package org.nd4j.parameterserver.distributed.v2.enums;
+
+import org.nd4j.linalg.exception.ND4JIllegalStateException;
+
+/**
+ * This enum describes possible result codes for Aeron-powered transmission
+ *
+ * @author raver119@gmail.com
+ */
+public enum TransmissionStatus {
+    UNKNOWN,
+    OK,
+    NOT_CONNECTED,
+    BACK_PRESSURED,
+    ADMIN_ACTION,
+    CLOSED,
+    MAX_POSITION_EXCEEDED;
+
+
+    public static TransmissionStatus fromLong(long value) {
```

Comment:
I note there's a fromLong, but no toLong, i.e., the enum/long relationship isn't defined anywhere except this method

Reply:
there's no need of toLong, because long value comes from Aeron. We don't originate this value anywhere.